[Bug] test_flashattn_backend and test_flashattn_mla_backend failed due to multiple missing parameters

### Checklist

- [x] I searched related issues but found no solution.
- [x] The bug persists in the latest version.
- [x] Issues without environment info and a minimal reproducible demo are hard to resolve and may receive no feedback.
- [x] If this is not a bug report but a general question, please start a discussion at https://github.com/sgl-project/sglang/discussions. Otherwise, it will be closed.
- [x] Please use English. Otherwise, it will be closed.

### Describe the bug

## Bug Description
Multiple tests fail in `test_flashattn_backend` and `test_flashattn_mla_backend`.

## Error Message

Result for `test_flashattn_backend`
```bash
FAILED python/sglang/test/attention/test_flashattn_backend.py::TestFlashAttentionBackend::test_forward_decode - Exception: retry() exceed maximum number of retries.
FAILED python/sglang/test/attention/test_flashattn_backend.py::TestFlashAttentionBackend::test_forward_decode_with_page_size_greater_than_1 - Exception: retry() exceed maximum number of retries.
FAILED python/sglang/test/attention/test_flashattn_backend.py::TestFlashAttentionBackend::test_forward_extend - Exception: retry() exceed maximum number of retries.
FAILED python/sglang/test/attention/test_flashattn_backend.py::TestFlashAttentionBackend::test_forward_extend_with_page_size_greater_than_1 - Exception: retry() exceed maximum number of retries.
FAILED python/sglang/test/attention/test_flashattn_backend.py::TestFlashAttentionBackend::test_forward_extend_with_prefix - Exception: retry() exceed maximum number of retries.
======================================================== 5 failed, 2 passed, 4 warnings in 13.99s =========================================================
```

Result for `test_flashattn_mla_backend`

```bash
FAILED python/sglang/test/attention/test_flashattn_mla_backend.py::TestFlashAttentionMLABackend::test_forward_decode - AttributeError: type object 'ModelConfig' has no attribute 'is_encoder_decoder'
FAILED python/sglang/test/attention/test_flashattn_mla_backend.py::TestFlashAttentionMLABackend::test_forward_extend - AttributeError: type object 'ModelConfig' has no attribute 'is_encoder_decoder'
FAILED python/sglang/test/attention/test_flashattn_mla_backend.py::TestFlashAttentionMLABackend::test_forward_extend_with_prefix - AttributeError: type object 'ModelConfig' has no attribute 'is_encoder_decoder'
============================================================== 3 failed, 2 warnings in 6.66s ==============================================================
```




### Reproduction


## Steps to Reproduce
```bash
cd sglang
python -m pytest python/sglang/test/attention/test_flashattn_backend.py -v
python -m pytest python/sglang/test/attention/test_flashattn_mla_backend.py -v
```

### Environment

## Environment

```bash
Python: 3.12.3 (main, Jan  8 2026, 11:30:50) [GCC 13.3.0]
CUDA available: True
GPU 0: NVIDIA H100 80GB HBM3
GPU 0 Compute Capability: 9.0
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 12.9, V12.9.86
CUDA Driver Version: 575.57.08
PyTorch: 2.9.1+cu129
sglang: 0.5.8
sgl_kernel: 0.3.21
flashinfer_python: 0.6.2
flashinfer_cubin: 0.6.2
flashinfer_jit_cache: 0.6.2+cu129
triton: 3.5.1
transformers: 4.57.1
torchao: 0.9.0
numpy: 2.4.1
aiohttp: 3.13.3
fastapi: 0.128.0
hf_transfer: 0.1.9
huggingface_hub: 0.36.0
interegular: 0.3.3
modelscope: 1.34.0
orjson: 3.11.5
outlines: 0.1.11
packaging: 26.0
psutil: 7.2.1
pydantic: 2.12.5
python-multipart: 0.0.21
pyzmq: 27.1.0
uvicorn: 0.40.0
uvloop: 0.22.1
vllm: Module Not Found
xgrammar: 0.1.27
openai: 2.6.1
tiktoken: 0.12.0
anthropic: 0.76.0
litellm: Module Not Found
decord2: 3.0.0
NVIDIA Topology: 
        GPU0    NIC0    NIC1    NIC2    NIC3    NIC4    NIC5    NIC6    NIC7    NIC8    NIC9    CPU Affinity    NUMA Affinity   GPU NUMA ID
GPU0     X      SYS     SYS     SYS     SYS     SYS     SYS     NODE    NODE    PXB     PXB     56-111,168-223  1               N/A
NIC0    SYS      X      PXB     NODE    NODE    NODE    NODE    SYS     SYS     SYS     SYS
NIC1    SYS     PXB      X      NODE    NODE    NODE    NODE    SYS     SYS     SYS     SYS
NIC2    SYS     NODE    NODE     X      PXB     NODE    NODE    SYS     SYS     SYS     SYS
NIC3    SYS     NODE    NODE    PXB      X      NODE    NODE    SYS     SYS     SYS     SYS
NIC4    SYS     NODE    NODE    NODE    NODE     X      PIX     SYS     SYS     SYS     SYS
NIC5    SYS     NODE    NODE    NODE    NODE    PIX      X      SYS     SYS     SYS     SYS
NIC6    NODE    SYS     SYS     SYS     SYS     SYS     SYS      X      PXB     NODE    NODE
NIC7    NODE    SYS     SYS     SYS     SYS     SYS     SYS     PXB      X      NODE    NODE
NIC8    PXB     SYS     SYS     SYS     SYS     SYS     SYS     NODE    NODE     X      PXB
NIC9    PXB     SYS     SYS     SYS     SYS     SYS     SYS     NODE    NODE    PXB      X 

Legend:

  X    = Self
  SYS  = Connection traversing PCIe as well as the SMP interconnect between NUMA nodes (e.g., QPI/UPI)
  NODE = Connection traversing PCIe as well as the interconnect between PCIe Host Bridges within a NUMA node
  PHB  = Connection traversing PCIe as well as a PCIe Host Bridge (typically the CPU)
  PXB  = Connection traversing multiple PCIe bridges (without traversing the PCIe Host Bridge)
  PIX  = Connection traversing at most a single PCIe bridge
  NV#  = Connection traversing a bonded set of # NVLinks

NIC Legend:

  NIC0: mlx5_0
  NIC1: mlx5_1
  NIC2: mlx5_2
  NIC3: mlx5_3
  NIC4: mlx5_4
  NIC5: mlx5_5
  NIC6: mlx5_6
  NIC7: mlx5_7
  NIC8: mlx5_8
  NIC9: mlx5_9


ulimit soft: 4096
```

--- a/python/sglang/test/attention/test_flashattn_backend.py
+++ b/python/sglang/test/attention/test_flashattn_backend.py
@@ -24,6 +24,9 @@ class MockModelRunner:
     ):
         self.device = "cuda"
         self.dtype = torch.float16
+        self.kv_cache_dtype = torch.float16
+        self.is_hybrid_swa = False
+        self.attention_chunk_size = None
         attention_arch = AttentionArch.MHA
         # Max batch size for the test.
         max_batch_size = 160
@@ -36,10 +39,11 @@ class MockModelRunner:
                 "context_len": max_context_len,
                 "is_multimodal": False,
                 "attention_arch": attention_arch,
+                "is_encoder_decoder": False,
+                "is_local_attention_model": False,
             },
-        )
+        )()
         self.sliding_window_size = None
-        self.device = self.device
         # Create a large enough req_to_token_pool to fit the test usage.
         self.req_to_token_pool = type(
             "TokenPool",
@@ -55,7 +59,7 @@ class MockModelRunner:
                     device=self.device,
                 ),
             },
-        )
+        )()
         self.page_size = page_size
         max_total_num_tokens = max_batch_size * max_context_len
         self.token_to_kv_pool = MHATokenToKVPool(

--- a/python/sglang/test/attention/test_flashattn_mla_backend.py
+++ b/python/sglang/test/attention/test_flashattn_mla_backend.py
@@ -28,6 +28,8 @@ class MockModelRunner:
             {
                 "context_len": context_len,
                 "attention_arch": attention_arch,
+                "is_encoder_decoder": False,
+                "is_local_attention_model": False,
             },
         )
         self.sliding_window_size = None

